x/concentrated-liquidity/types/msgs_test.go

```diff
@@ -41,9 +44,13 @@ func runValidateBasicTest(t *testing.T, name string, msg extMsg, expectPass bool
 		require.NoError(t, msg.ValidateBasic(), "test: %v", name)
 		require.Equal(t, msg.Route(), types.RouterKey)
 		require.Equal(t, msg.Type(), expType)
-		signers := msg.GetSigners()
+		encCfg := osmosisapp.GetEncodingConfig().Marshaler
+		signers, _, err := encCfg.GetMsgV1Signers(msg)
+		if err != nil {
+			require.NoError(t, err)
+		}
```

Comment:
nit: this check is odd

Reply:
lol super weird, changed here https://github.com/osmosis-labs/osmosis/pull/8274/commits/117194aab651ddcaf42c2e6bfa16964805498ac8